docs/apache-airflow/concepts/dags.rst

```diff
@@ -796,39 +796,44 @@ the dependency graph.
-``DAGS_FOLDER``. When scheduler parses the ``DAGS_FOLDER`` and misses the DAG that it had seen
-before and stored in the database it will set is as deactivated. The metadata and history of the
-DAG` is kept for deactivated DAGs and when the DAG is re-added to the ``DAGS_FOLDER`` it will be again
-activated and history will be visible. You cannot activate/deactivate DAG via UI or API, this
-can only be done by removing files from the ``DAGS_FOLDER``. Once again - no data for historical runs of the
-DAG are lost when it is deactivated by the scheduler. Note that the ``Active`` tab in Airflow UI
-refers to DAGs that are not both ``Activated`` and ``Not paused`` so this might initially be a
-little confusing.
-
-You can't see the deactivated DAGs in the UI - you can sometimes see the historical runs, but when you try to
-see the information about those you will see the error that the DAG is missing.
-
-You can also delete the DAG metadata from the metadata database using UI or API, but it does not
-always result in disappearing of the DAG from the UI - which might be also initially a bit confusing.
-If the DAG is still in ``DAGS_FOLDER`` when you delete the metadata, the DAG will re-appear as
-Scheduler will parse the folder, only historical runs information for the DAG will be removed.
-
-This all means that if you want to actually delete a DAG and its all historical metadata, you need to do
-it in three steps:
-
-* pause the DAG
-* delete the historical metadata from the database, via UI or API
-* delete the DAG file from the ``DAGS_FOLDER`` and wait until it becomes inactive
+Controlling DAG Execution
+--------------------------
+
+A DAG can be prevented from running by being paused, deactivated, or deleted.
+
+Pausing a DAG
+^^^^^^^^^^^^^
+
+Pausing a DAG will temporarily prevent scheduled runs from occurring.
+The Scheduler will not schedule a paused DAG, but you can still trigger it manually
+via the UI. The "pause" and "unpause" actions are available in the UI and through the API.
+In the UI, paused DAGs can be found in the ``Paused`` tab, and unpaused DAGs can be found in the 
+``Active`` tab.
+
+Deactivating a DAG
+^^^^^^^^^^^^^^^^^^
+
+Deactivating a DAG will prevent it from being scheduled or triggered, and it will not appear in the UI.
+However, its metadata and history will still be persisted in the database.
+When the DAG is reactivated, all of its history and metadata will still be present.
+
+To deactivate a DAG, remove it from the ``DAGS_FOLDER``.
+When the Scheduler finds a DAG in the database but missing from the ``DAGS_FOLDER``,
+it will mark it as deactivated. The metadata and history of the DAG will be kept.
+If the DAG is added back to the ``DAGS_FOLDER``, the Scheduler will mark it as active
+and its history will be visible again. Note that the "Active" tab in the Airflow UI refers
+to DAGs that are not deactivated and not paused.
```

Comment:
```suggestion
To remove a DAG from the UI, remove the corresponding DAG file. Airflow calls such DAGs "deactivated" DAGs. When the scheduler detects a DAG file is missing from the ``DAGS_FOLDER`` it will hide the corresponding DAG in the UI. All metadata and history of the DAG will be preserved. When the DAG file is added back to the ``DAGS_FOLDER``, the DAG and all corresponding metadata and history will be displayed again in the UI.

A "deactivated" DAG cannot be scheduled or triggered.
```

"active" and "deactivated" are internal states and the user never gets to see these. Would focus on the behaviour from a user's perspective.